Test/TestParser/TestSay/TestSay.gd

```diff
@@ -51,27 +51,7 @@ func test_say_character():
 	
 	assert_eq(result["text"], "Hello !")
 
-func test_say_variable():
-	var rk_script = [
-		"\"My name is <sy.name>\""
-	]
 
-	var parsed_script = parser.parse_script(rk_script)
-
-	assert_false(parsed_script.is_empty())
-
-	var parsed_array = parsed_script["parse_array"]
-
-	assert_false(parsed_array.is_empty())
-	
-	assert_true(parsed_script.get("labels", [""]).is_empty())
-
-	assert_true(parsed_array[0][0] == "SAY")
-
-	var result = parsed_array[0][1]
-	
-	assert_eq(result["text"], "My name is <sy.name>")
-	
```

Comment:
Why do you remove it ? It is permit to certified the parser recognize string with variable and do not broke them

Reply:
No, it dosen't do this as Parser doesn't handel variables at strings at all, Executer dose that. So it can react to change of variable values in run time.

Reply:
Yeah, it does not handle the replace of the variable, but parser parse. And we need to test it, to be sure it work as well intented and produce a result that can be used by the executer.

Reply:
Okey, I will restore it.

Reply:
Restored :grin:

Reply:
Outdated, I put it to reloved